.github/workflows/run_test_suite.yaml

```diff
@@ -55,7 +55,18 @@ jobs:
           ipfs_version: v0.17.0
           run_daemon: true
       - name: 'Run Rust native target tests'
-        run: cargo test --features test-kubo,helpers
+        # Increase stack size on Windows tests; (non-main) threads are spawned with 2MB
+        # default stack size, which `orb_can_render_peers_in_the_sphere_address_book`
+        # uses (at time of writing) slightly more than 2MB. While we could set the thread
+        # stack size at runtime (via tokio's `thread_stack_size`), it appears to not solve the
+        # problem, possibly due to the harness thread overflowing (e.g. a non-main thread that
+        # we can't configure within the test). In lieu of that, set RUST_MIN_STACK to increase
+        # the stack sizes of threads created by tokio within tests, as well as the test harness
+        # threads themselves.
+        #
+        # While our main thread isn't under fire here, notating this for future use:
+        # https://users.rust-lang.org/t/stack-overflow-when-compiling-on-windows-10/50818/8
```

Comment:
Ideally, this would be applied when running locally outside of CI.

The immediate solutions to achieve this would be to increase the stack size of only the offending test, but the test harness thread itself is overflowing it seems. Alternatively, a Cargo.toml test-only flag such that would apply in both CI and local runs, but no leads yet on how to configure something like that.